src/main/java/de/retest/suite/flow/ApplyChangesToStatesFlow.java

```diff
@@ -0,0 +1,74 @@
+package de.retest.suite.flow;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.retest.persistence.NoStateFileFoundException;
+import de.retest.persistence.Persistence;
+import de.retest.persistence.RecheckStateFileProvider;
+import de.retest.persistence.RecheckStateFileProviderImpl;
+import de.retest.ui.descriptors.SutState;
+import de.retest.ui.review.ActionChangeSet;
+import de.retest.ui.review.SuiteChangeSet;
+import de.retest.ui.review.TestChangeSet;
+
+public class ApplyChangesToStatesFlow {
+
+	private static final Logger logger = LoggerFactory.getLogger( ApplyChangesToStatesFlow.class );
+
+	private final RecheckStateFileProvider recheckStateFileProvider;
+
+	public static List<String> apply( final Persistence<SutState> persistence, final SuiteChangeSet acceptedChanges )
```

Comment:
Should be `applyChangesToStates` if we follow the naming scheme in `CreateChangesetForAllDifferencesFlow`.

Reply:
Or we stick to `apply`, but then it should be `CreateChangesetForAllDifferencesFlow#create(...)`.